.github/workflows/integration.yml

```diff
@@ -149,7 +158,7 @@ jobs:
       - name: Create k8s cluster for testing
         run: |
           doctl kubernetes cluster create ${{ env.DIGITALOCEAN_CLUSTER_NAME }} \
-            --tag integration-test \
+            --tag source:github \
```

Comment:
Maybe we can decide on a tag for temporary resources so we can run automated cleanup

Reply:
Good point, I want to create a Github pipeline to remove all clusters with certain tags, so if someone forget removing adhoc cluster then it gets deleted at night anyway. TBD as a separate PR